symplyphysics/laws/optics/linear_magnification_from_distance_to_object_and_distance_to_image.py

```diff
@@ -0,0 +1,40 @@
+from sympy import (Eq, solve, S)
+from symplyphysics import (units, Quantity, Symbol, print_expression, validate_input,
+                           validate_output, dimensionless, convert_to)
+
+# Description
+## Magnification, in optics, the size of an image relative to the size of the object creating it.
+## Depending on the position of the object in relation to the lens, the linear dimensions of the image change.
+
+
+## Law: M = f / d
+## Where:
+## M is linear magnification produced by lense
+## f is distance from lens to image
+## d is distance from lens to object
+
+## Conditions
+## If the image is straight, then a plus sign is placed before the value f and a minus sign if the image is inverted.

```

Comment:
It seems you have inverted signs, when compared to common convention. Try to use the same convention: https://www.teachoo.com/10842/3118/Lens-Formula/category/Concepts/